benchmark/torch/ddpg/mujoco_model.py

```diff
@@ -0,0 +1,71 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import parl
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+'''
+Model of DDPG:  defines an Actor/policy network given obs as input,
+                      & a Critic/value network given obs and action as input.
+'''
+
+
+class MujocoModel(parl.Model):
+    def __init__(self, obs_dim, action_dim, max_action):
+        super(MujocoModel, self).__init__()
+        self.actor_model = Actor(obs_dim, action_dim, max_action)
```

Comment:
Do not pass max_action here. Mapping the action into the value range of the environment can be finished in the environment wrapper.
Please refer to the implementation in TD3.
https://github.com/PaddlePaddle/PARL/blob/8877d68f87407c438ba079cd8d56303dfae95521/examples/TD3/train.py#L23

Reply:
done